src/main/java/mrriegel/storagenetwork/block/cable/BlockCable.java

```diff
@@ -180,11 +228,55 @@ public boolean onBlockActivated(World worldIn, BlockPos pos, IBlockState state,
     return false;
   }
 
+  @SuppressWarnings("deprecation")
+  @Override
+  public void onBlockClicked(World worldIn, BlockPos pos, EntityPlayer playerIn) {
+    if (!playerIn.isSneaking()) {
+      StorageNetwork.log("no sneak");
+      return;
+    }
+    TileCable tile = getTileCable(worldIn, pos);
+    if (tile == null) {
+      StorageNetwork.log("no tile");
+      return;
+    }
+    ItemStack heldStack = playerIn.getHeldItemMainhand();
+    if (heldStack == null || heldStack.isEmpty()) {
+      tile.facadeState = null;
+      worldIn.markAndNotifyBlock(pos.toImmutable(), worldIn.getChunk(pos), getDefaultState(), getDefaultState(), 1 | 2);
+      tile.markDirty();
+      StorageNetwork.log("unset facade");
+      return;
+    }
+    Block block = Block.getBlockFromItem(heldStack.getItem());
+    if (block == null || block == Blocks.AIR) {
+      StorageNetwork.log("no block");
+      return;
+    }
+    int meta = heldStack.getMetadata();
+    RayTraceResult mouseOver = Minecraft.getMinecraft().objectMouseOver;
+    float f = (float) (mouseOver.hitVec.x - (double) pos.getX());
+    float f1 = (float) (mouseOver.hitVec.y - (double) pos.getY());
+    float f2 = (float) (mouseOver.hitVec.z - (double) pos.getZ());
+    IBlockState state = block.getStateForPlacement(worldIn, pos, mouseOver.sideHit, f, f1, f2, meta, playerIn);
+    if (state == null || state.getRenderType() != EnumBlockRenderType.MODEL) {
+      StorageNetwork.log("no model");
+      return;
+    }
+    tile.facadeState = state;
+    worldIn.markAndNotifyBlock(pos.toImmutable(), worldIn.getChunk(pos), getDefaultState(), getDefaultState(), 1 | 2);
+    tile.markDirty();
+    StorageNetwork.log("set facade");
+  }
+
   @SuppressWarnings("deprecation")
   @Override
   public void addCollisionBoxToList(IBlockState state, World worldIn, BlockPos pos, AxisAlignedBB entityBox, List<AxisAlignedBB> collidingBoxes, @Nullable Entity entityIn, boolean p_185477_7_) {
-    TileEntity tileHere = worldIn.getTileEntity(pos);
-    if (!(tileHere instanceof TileCable)) {
+    TileCable tile = getTileCable(worldIn, pos);
+    if (tile == null)
+      return;
+    if (tile.facadeState != null) {
+      tile.facadeState.addCollisionBoxToList(worldIn, pos, entityBox, collidingBoxes, entityIn, p_185477_7_);
```

Comment:
TODO: maybe a global config file setting, to say "facadesUseCollisionBox: true/false" . give players more control.  see: flowers as facades